clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/KubernetesApiVersion.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.v2.description;
+
+import java.util.Arrays;
+
+public enum KubernetesApiVersion {
+  V1("v1"),
+  EXTENSIONS_V1BETA1("extensions/v1beta1");
```

Comment:
Does this correspond to v2 of the Kubernetes provider? Will there be a mapping defined between Kubernetes API version and provider version?

Reply:
No these are Kubernetes-only release tracks. The client library is broken down by operations on each of these, so we need to be able to pull the right release track from a manifest to know how to update/deploy/destroy it.

We want to support all of these, so no mapping is really needed. What is important will be to connect Spinnaker versions to Kubernetes versions (both separate from provider version and kubernetes api version entirely).